manual/manual/tutorials/coreexamples.etex

```diff
@@ -677,6 +677,36 @@ print_expr e; print_newline ();;
 print_expr (deriv e "x"); print_newline ();;
 \end{caml_example}
 
+There is a "printf" function in "Printf" module that allows you to make
+formatted output more concisely. It models the behavior of the "printf"
+function from the C standard library.
+The "printf" function takes a format string that describes the output values and
+their format, and arguments for output that correspond to the formatting
+directives in the format string.
+\begin{caml_eval}
+open Printf;;
+\end{caml_eval}
+\begin{caml_example}{toplevel}
+printf "%i + %i is an integer value, %F * %F is a float, \"%s\"\n"
+3 2 4.5 1. "this is a string";;
+\end{caml_example}
+The printf function in the OCaml language performs type checking of arguments.
+For example, if you pass it an argument of a type that does not correspond to
+the format specifier, the compiler will display an error message:
+\begin{caml_example}{toplevel}[error]
+printf "Float value: %F" 42;;
+\end{caml_example}
+Associated with this feature is the need to explicitly indicate that the
```

Comment:
It would be nice, if possible, to link to the `Printf` module page and the section discussin the conversion specification and the types available (see https://caml.inria.fr/pub/docs/manual-ocaml/libref/Printf.html)

Reply:
This is beyond my own skills with manual references. @Octachron, do you remember whether we can link from the manual to the stdlib documentation? (We should describe how to do this in the manual README.)

Reply:
It is possible, but the link target is  backend dependent. It is probably simpler that I add a latex macro to take care of those differences.

Reply:
@Octachron If I use the \href tag and add a link to the specified documentation page via it, would this be an acceptable solution? Or should it be postponed?

Reply:
We can start with a `href` and update it later indeed.